app/build.gradle

```diff
@@ -51,9 +51,7 @@ dependencies {
     androidTestImplementation "com.android.support:support-annotations:${project.supportLibVersion}"
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
 
-    debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.5.1'
-    releaseImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.1'
-    testImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.1'
+    implementation 'com.squareup.leakcanary:leakcanary-android:1.5.1'
```

Comment:
What is the purpose of this change? It appears to make the release build larger with the leakcanary classes which are not used.